sentential/lib/shapes.py

```diff
@@ -14,7 +14,10 @@
 #
 
 SNTL_WORKING_IMAGE_TAG = os.getenv("SNTL_WORKING_IMAGE_TAG", default=("cwi"))
-SNTL_ENTRY_VERSION = os.getenv("SNTL_ENTRY_VERSION", default=("0.4.1"))
+SNTL_ENTRY_VERSION = os.getenv("SNTL_ENTRY_VERSION", default=("0.4.2"))
+SNTL_ENTRY_PATHS = os.getenv(
+    "SNTL_ENTRY_PATHS", default=("${PARTITION}/Envs,${PARTITION}/Secrets")
```

Comment:
should `${PARTITION}` here actually reference our `Ontology().context`? I'm very undecided on this. Methinks you did this quite intentionally.

Reply:
That makes most sense I think!

Tossed this together as I ran out the door, when it was still present in the **Dockerfile**.

Reply:
Though, if derived from `${PARTITION}', the environment variable could survive an out-of-band (sntl-less) modification of the partition variable itself.

I don't think it's our responsibility to account for that however :)